assets/js/hocs/with-attributes.js

```diff
@@ -30,52 +29,54 @@ const getAttributeData = ( attributeId, attributeList, matchField = 'id' ) => {
  */
 const withAttributes = ( OriginalComponent ) => {
 	return ( props ) => {
-		const { selected } = props;
-		const [ attributes, setAttributes ] = useState( [] );
+		const { selected = [] } = props;
+		const selectedSlug = selected.length ? selected[ 0 ]?.attr_slug : null;
```

Comment:
In my experience, when working with an expected data endpoint (like our own REST API), using optional chaining signals a code smell.
Disregarding line 32, do we expect `selected` to be an empty array?
Also if you defaulting to an empty array in line 32, `length` would evaluate to 0, which would skip directly to null, so no optional chaining required.

Reply:
Yeah... good point. I don't think the optional chaining is necessary at all, probably it's a left-over from an in-progress refactor I was doing. Fixed in bca57a5afe2af6d03834e0e3bd945abba1369482.